.github/workflows/opam-build.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           # - windows-latest # for some reason windows build is not picking dune 3.5
         ocaml-compiler:
           - ocaml-base-compiler.5.1.1
+          # - 'ocaml-base-compiler.5.2.0~alpha1'
```

Comment:
Will this be uncommented before merging?

Reply:
No. The compiler libs submodule is on 5.1 according to the proposal I shared: have melange remain on 5.1 but ready to integrate the 5.2 submodule once OCamlformat and odoc have compatible releases.